File: java-junit/src/main/java/org/codecop/Greeter.java
package org.codecop;

public class Greeter {

    private static final String GREETING = "Hello";

    /**
     * Return a proper greeting for a person.
     */
    public String greet(String name) {
        if (name == null) {
            return GREETING;
        }
        return GREETING + " " + name.trim();
    }
}

File: csharp-xunit/WordCount/Greeter.cs
namespace Org.Codecop.WordCount
{
    public class Greeter
    {
        private const string Greeting = "Hello";

        /// <summary>Return a proper greeting for a person.</summary>
        public string Greet(string name)
        {
            if (name == null)
            {
                return Greeting;
            }
            return Greeting + " " + name.Trim();
        }
    }
}
